src/main/java/com/dev/cinema/service/mapper/TicketMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.dto.MovieSessionResponseDto;
+import com.dev.cinema.model.dto.TicketResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TicketMapper {
```

Comment:
I think we can remove this

we will return ticketIds
if necessary, the additional info about tickets can be fetched given these ids